lib/agent.js

```diff
@@ -6,6 +6,10 @@
  * See the included LICENSE file for more details.
  */
 
+// Polyfill for `String.prototype.normalize`.
+// To be removed when Node 0.x is no longer supported.
+require('unorm')
```

Comment:
Only Node 0.10 is affected; since it is almost EOL, maybe support could just be dropped for the next release.